designs/tripleo/osc-tripleo.md

```diff
@@ -176,6 +183,30 @@ path/directory on overcloud node and the TLS certificate is copied to this locat
 
 >>c. Do we need to hard code the target path for storing the certificate? If so how is this used by puppet?
 
+#### OSC Network Configuration
+1. Add an entry in the file `network/service_net_map.j2.yaml` under the `ServiceNetMapDefaults` section for `OSCNetwork` like below. This is required to prevent OSC service to be part of provisioning network.
+    ```yaml
+        ServiceNetMapDefaults:
+            default:
+                ...
+                OSCNetwork: internal_api
+    ```
+    >Assumption: OSC Service needs to be placed under internal_api network by default.
+
+2. Add an entry in the file `network/endpoints/endpoint_data.yaml` for OSC like below. This will create a public endpoint for OSC service using which user can access the OSC UI.
+    ```yaml
+        OSC:
+            Public:
+                net_param: Public
+                uri_suffixes:
+                    '': /osc_dashboard
+                port: 443
```

Comment:
is this the host port? I think the 443 port is generally used for the primary service like horizon in openstack etc.

we need to be able to expose a different port for support services like OSC etc

How do we map any host port to one of the OSC ports within the container?

Reply:
Yes, these two things I have added as part of TODO items.
For the first one we need to decide a good number which we can use as a port for OSC service.
Here, I have given one example how format will look like.
For the second one we may need to consult with RH.

Reply:
I believe we will need two endpoints here OSCDashboard and OSCAPI since we bind two ports. @sahasamiran you can leave it like this when you merge and i can update when i send my changes.

Reply:
Ok.. sure.